dist/stat_data.py

```diff
@@ -220,6 +220,10 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_dirty', 'modified pages evicted'),
     CacheStat('cache_eviction_empty_score', 'eviction empty score', 'no_clear,no_scale'),
     CacheStat('cache_eviction_fail', 'pages selected for eviction unable to be evicted'),
+    CacheStat('cache_eviction_fail_active_children_on_an_internal_page', 'pages selected for eviction unable to be evicted because of active children on an internal page'),
+    CacheStat('cache_eviction_fail_in_reconciliation', 'pages selected for eviction unable to be evicted because of failure in reconciliation'),
+    CacheStat('cache_eviction_fail_overflow_keys', 'pages selected for eviction unable to be evicted because of overflow keys'),
```

Comment:
Need to further elaborate the description of this stat.

Reply:
Improved the description of the stat with more details to understand.